converters/events_test.go

```diff
@@ -0,0 +1,48 @@
+package converters
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConvertTxsSliceIntoMap(t *testing.T) {
+	t.Parallel()
+
+	tx1 := &data.Transaction{
+		Hash:  "h1",
+		Nonce: 1,
+	}
+	tx2 := &data.Transaction{
+		Hash:  "h2",
+		Nonce: 2,
+	}
+	txsSlice := []*data.Transaction{tx1, tx2}
+	txsMap := ConvertTxsSliceIntoMap(txsSlice)
+
+	txsMap["h1"].Nonce = 2
```

Comment:
```GO
require.True(t, tx1 == txsMap["h1"]) //pointer testing
require.True(t, tx2 == txsMap["h2"]) //pointer testing
require.Equal(t, len(txsSlice), len(txsMap))
```

valid also for the TestConvertScrsSliceIntoMap function

Reply:
done.